ext/opentelemetry-ext-flask/src/opentelemetry/ext/flask/__init__.py

```diff
@@ -12,9 +12,9 @@
 
 logger = logging.getLogger(__name__)
 
-_ENVIRON_STARTTIME_KEY = object()
-_ENVIRON_SPAN_KEY = object()
-_ENVIRON_ACTIVATION_KEY = object()
+_ENVIRON_STARTTIME_KEY = "opentelemetry-flask.starttime_key"
```

Comment:
```suggestion
_ENVIRON_STARTTIME_KEY = "opentelemetry-flask.starttime"
```

I think we don't need to add `_key` to the key.